dynamic_array/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.11)
+
+add_definitions(-std=c++11)
```

Comment:
Why not `c++14` or 17?

Reply:
Currently, I try to make it work on c++11